lib/local-links-manager/import/services_tier_importer.rb

```diff
@@ -0,0 +1,93 @@
+require_relative 'csv_downloader'
+
+module LocalLinksManager
+  module Import
+    class ServicesTierImporter
+      CSV_URL = "https://raw.githubusercontent.com/alphagov/publisher/master/data/local_services.csv"
+      class MissingRecordError < RuntimeError; end
+      class MissingIdentifierError < RuntimeError; end
+
+      def self.import
+        new.import_tiers
+      end
+
+      def initialize(csv_downloader = CsvDownloader.new(CSV_URL))
+        @csv_downloader = csv_downloader
+        @csv_rows = 0
+        @missing_record_count = 0
+        @missing_id_count = 0
+        @invalid_record_count = 0
+        @updated_record_count = 0
+        @ignored_rows_count = 0
+      end
+
+      def import_tiers
+        with_each_csv_row do |row|
+          counting_errors do
+            if update_record(parsed_hash(row))
```

Comment:
This could be rewritten now since https://github.com/alphagov/local-links-manager/pull/12 got merged, making the `CsvDownloader` class do the field_name_conversions